docs/locust/README.md

```diff
@@ -72,7 +72,7 @@ $ curl -X POST http://${KANGAL_PROXY_ADDRESS}/load-test \
   -F type=Locust
 ```
 
-In this last example, the test will run infinitely and no `targetURL` were specified.
+In this last example, the test will run infinitely and no `targetURL` was specified.
```

Comment:
```suggestion
In this last example, the test will run infinitely and no `targetURL` is specified in request, since it's set in the test code.
```

Reply:
Updated in [112d17c](https://github.com/hellofresh/kangal/pull/251/commits/112d17c15adbb37c7fc1b4b7e810d2bd63c113fd).